attributes/default.rb

```diff
@@ -377,6 +377,12 @@
 default['hops']['hopsutil_version']                    = node['install']['version']
 default['hops']['hopsutil']['url']                     = "#{node['download_url']}/hopsutil/#{node['hops']['hopsutil_version']}/hops-util-#{node['hops']['hopsutil_version']}.jar"
 
+#
+# Tensorflow-Spark Connector (Featurestore dependency)
+#
+default['hops']['tf_spark_connector_version']                            = "2.11-1.12.0"
+default['hops']['tf_spark_connector']['url']                     = "#{node['download_url']}/spark-tensorflow-connector_#{node['hops']['tf_spark_connector_version']}.jar"
+
```

Comment:
Why didn't you put these in the `spark-chef` cookbook?

Reply:
@SirOibaf No particular reason, just followed the same practice as hops-util.jar and hops-spark-examples.jar, do you want me to move it?

Reply:
Prehistorical reason actually. I agree

Reply:
This is fixed now, I will close this PR, please add comments to spark-chef PR if you have more feedback.